modules/api/api-xsjs/src/main/resources/META-INF/dirigible/xsk/web/web.js

```diff
@@ -283,7 +288,7 @@ exports.WebRequest = function (method, path) {
     }.bind(this)();
 
     this.parameters = function () {
-      var dArrayOfPairs = dRequest.getParameters();
+      let dArrayOfPairs = dRequest.getParameters();
```

Comment:
Use `const`

Reply:
Value of dArrayOfPairs is later updated

Reply:
`const` means the variable can't be reassigned, not that you can't modify it. Refer to MDN for additional docs